pyproject.toml

```diff
@@ -11,6 +16,11 @@ profile = 'black'
 sections = ['FUTURE', 'STDLIB', 'THIRDPARTY', 'EXTERNAL_PACKAGE', 'FIRSTPARTY', 'LOCALFOLDER']
 skip_glob = ['download_and_create_reference_datasets/*', 'gcloud_dataproc/*', 'hail_scripts/*', 'kubernetes/*']
 
+[tool.mypy]
```

Comment:
Other benefit to just starting a new folder is we can get some foundational pieces like `mypy` in from the start.

Here's a mypy example from typo-ing a str for an enum:

```
bblanken@wmbbe-67c seqr-loading-pipelines % git diff
diff --git a/v03_pipeline/paths.py b/v03_pipeline/paths.py
index 4c3e543..c4fca0c 100644
--- a/v03_pipeline/paths.py
+++ b/v03_pipeline/paths.py
@@ -30,7 +30,7 @@ def project_table_path(
 ) -> str:
     return os.path.join(
         SEQR_SCRATCH_TEMP if env == Env.DEV else SEQR_DATASETS,
-        reference_genome.value,
+        reference_genome,
         V03,
         dataset_type.value,
         'projects',
bblanken@wmbbe-67c seqr-loading-pipelines % mypy
v03_pipeline/paths.py:31: error: Returning Any from function declared to return "str"  [no-any-return]
v03_pipeline/paths.py:31: error: No overload variant of "join" matches argument types "str", "ReferenceGenome", "str", "Union[str, str, str, str]", "str", "str", "str"  [call-overload]
v03_pipeline/paths.py:31: note: Possible overload variants:
v03_pipeline/paths.py:31: note:     def join(str, /, *paths: str) -> str
v03_pipeline/paths.py:31: note:     def join(Union[str, PathLike[str]], /, *paths: Union[str, PathLike[str]]) -> str
v03_pipeline/paths.py:31: note:     def join(Union[bytes, PathLike[bytes]], /, *paths: Union[bytes, PathLike[bytes]]) -> bytes
Found 2 errors in 1 file (checked 4 source files)
```